src/main/java/com/google/api/codegen/InterfaceConfig.java

```diff
@@ -76,18 +77,32 @@ public static InterfaceConfig createInterfaceConfig(
       methodConfigs = createMethodConfigs(methodConfigMap, interfaceConfigProto);
     }
 
+    SmokeTestConfig smokeTestConfig =
+        createSmokeTestConfig(diagCollector, iface, interfaceConfigProto);
+
```

Comment:
There needs to be a check for smokeTestConfig being null (it needs to be one of the conditions in the 'if' below)